apis/v1alpha1/gateway_types.go

```diff
@@ -483,15 +476,17 @@ type GatewayStatus struct {
 	//
 	// These addresses should all be of type "IPAddress".
 	//
-	// +required
+	// +kubebuilder:validation:MaxItems=16
 	Addresses []GatewayAddress `json:"addresses"`
 
 	// Conditions describe the current conditions of the Gateway.
 	// +optional
+	// +kubebuilder:validation:MaxItems=16
```

Comment:
MaxItems=8

Reply:
Updated max items to 8. Wouldn't `MinItems=0` be default? Do we need to specify it everywhere?